src/api/AccessRight/types.ts

```diff
@@ -22,6 +22,7 @@ export enum Action {
   RUN_QUERY = 'runQuery', // Execute a saved query
   RAW_READ_QUERY = 'rawReadQuery', // Create a read query
   RAW_WRITE_QUERY = 'rawWriteQuery', // Create a read/write query
+  RAW_MANAGE_QUERY = 'rawManageQuery', // Manage read/write queries
```

Comment:
I don't think the `raw` prefix is needed here. Also worth enriching a bit the documentation:

```suggestion
  MANAGE_QUERY = 'manageQuery', // Manage all non-private read/write queries
```

Reply:
> I don't think the `raw` prefix is needed here. Also worth enriching a bit the documentation:

it's a bit late and would require multiple changes in multiple repos when we are about to merge. I can open a ticket and do it next sprint...good for you?